code-challenges.js

```diff
@@ -41,18 +112,74 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
-
-
+//thoughts:
+    //We also did this in pig latin and office hours yesterday...
+
+//create a function that takes in a string
+const firstVowelIndex = (string) => {
+    //split the string into an array of characters
+    let charArr = string.split("")
+    //filter for vowels and return index (need filter method that takes value and index)
+        //using map instead, filter returns the values but not the indexes but map does for some reason (would like to know why)
+    let vowelOrNot = charArr.map((value, index) => {
+        //checks if value is a vowel, if it is then return the index
+        if (value === "a" || value === "e" || value === "i" || value === "o" || value === "u" || value === "A" || value === "E" || value === "I" || value === "O" || value === "U" )
```

Comment:
I think the only thing I would do is utilize .toLowerCase() on each .value before doing its comparison to keep you from having to repeat yourself multiple times to check those edge case possibilities. 
Like so:
`if (value.toLowerCase() === "a") would temporarily change the value to a lowercase so that no matter if its uppercase or lowercase value, it will force it into what you're checking for.